From 988a9a0570baec19a2a6403c11313ab65d08cd05 Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Tue, 27 Dec 2011 19:28:48 +0800
Subject: [PATCH 261/505] ENGR00170971: mlb test code can't find head file

mlb test code can't find mxc_mlb.h.
So fix it.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 test/mxc_mlb_test/mxc_mlb150_test.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/test/mxc_mlb_test/mxc_mlb150_test.c b/test/mxc_mlb_test/mxc_mlb150_test.c
index 285fde0..13b5683 100644
--- a/test/mxc_mlb_test/mxc_mlb150_test.c
+++ b/test/mxc_mlb_test/mxc_mlb150_test.c
@@ -25,8 +25,7 @@
 #include <errno.h>
 #include <string.h>
 
-//#include <linux/mxc_mlb.h>
-#include "mxc_mlb.h"
+#include <linux/mxc_mlb.h>
 
 #define MLB_SYNC_DEV	"/dev/sync"
 #define MLB_CTRL_DEV	"/dev/ctrl"
-- 
1.8.0

